prow/config.yaml

```diff
@@ -352,21 +352,15 @@ presubmits:
     rerun_command: "/test pull-kubernetes-e2e-gce-gci"
     trigger: "(?m)^/test pull-kubernetes-e2e-gce-gci,?(\\s+|$)"
   - name: pull-kubernetes-e2e-gce-gpu
-    agent: jenkins
-    always_run: false
-    context: pull-kubernetes-e2e-gce-gpu
-    rerun_command: "/test pull-kubernetes-e2e-gce-gpu"
-    trigger: "(?m)^/test pull-kubernetes-e2e-gce-gpu,?(\\s+|$)"
-  - name: pull-kubernetes-e2e-gce-gpu-prow
     agent: kubernetes
     skip_branches:
     - release-1.4
     - release-1.5
     - release-1.6
-    always_run: false
-    context: pull-kubernetes-e2e-gce-gpu-prow
-    rerun_command: "/test pull-kubernetes-e2e-gce-gpu-prow"
-    trigger: "(?m)^/test pull-kubernetes-e2e-gce-gpu-prow,?(\\s+|$)"
+    always_run: true
```

Comment:
add a `maximum_concurrency`

Reply:
done, also fixed removing it from jenkins and the regex having all. :roll_eyes: